appstudio-controller/controllers/appstudio.redhat.com/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -272,6 +274,15 @@ func generateExpectedGitOpsDeployment(component appstudioshared.ComponentStatus,
 		}
 	}
 
+	res.ObjectMeta.Labels = make(map[string]string)
+
+	// Append ASEB labels with prefix "appstudio.openshift.io" to the gitopsDeployment labels
+	for bindingKey, bindingLabels := range binding.Labels {
+		if strings.HasPrefix(bindingKey, "appstudio.openshift.io") {
```

Comment:
Let's move `"appstudio.openshift.io"` into a constant, and reference that constant here and in the tests.